users/tests.py

```diff
@@ -11,13 +11,18 @@
 from plio.cache import get_cache_key
 
 
-class OtpAuthTestCase(BaseTestCase):
+class OtpAuthTestCase(BaseTestCase):  
     """Tests the OTP functionality."""
 
     def setUp(self):
         super().setUp()
         # unset client credentials token so that the subsequent API calls goes as guest
         self.client.credentials()
+    def test_request_otp_mobile_not_passed(self):
```

Comment:
leaving a few similar feedback points as I mentioned on Manya's PR: the name of the test should specify what should happen given a particular set of inputs. So, in this case, a better name could be: `test_requesting_otp_fails_when_mobile_not_passed`

Reply:
corrected it